src/js/common/components/Modal.jsx

```diff
@@ -78,7 +78,7 @@ class Modal extends React.Component {
     }
 
     return (
-      <div className={modalCss} id={id} role="alertdialog" aria-labelledby={`${id}-title`}>
+      <div className={modalCss} id={id} role="alertdialog" aria-labelledby={id && title ? `${id}-title` : undefined}>
```

Comment:
Well, it should be undefined, and React shouldn't include the attribute at all in the html. But it looks like if you use the `alertdialog` role, you should have a label.

For what it's worth, we did run through all the modals on claim status with the 508 office, and made fixes, but this particular issue didn't come up.